src/components/functions.js

```diff
@@ -1,10 +1,164 @@
-import axios from 'axios';
-import {BASE_URL,OPTIMIZE_SVG} from '../constants/constants';
-
-export default  function optimizeSvg(dataUrl){
-    return axios.post(`${BASE_URL}${OPTIMIZE_SVG}`, { dataUrl })
-        .then(res => {
-            return res.data;
-        })
-        .catch(err => console.log(err));
-}
+import axios from "axios";
+import React from "react";
+import DisplaySvg from "../components/displaySvg/displaySvg";
+import finalSvgDisplay from "../components/displaySvg/finalSvgDisplay.scss";
+import { BASE_URL, OPTIMIZE_SVG } from "../constants/constants";
+import base64 from "base-64";
+
+export const optimizeSvg = (dataUrl, svgo) => {
+  return axios
+    .post(`${BASE_URL}${OPTIMIZE_SVG}`, { dataUrl, svgo })
+    .then(res => {
+      return res.data.urlData;
+    })
+    .catch(err => console.log(err));
+};
+
+export const changeObj = (value, svgoObjectPlugins) => {
+  let newSvgoObject = svgoObjectPlugins.filter(option => {
+    if (option[value] !== null && option[value] !== undefined) {
+      option[value] = true;
+    }
+    return true;
+  });
+  return newSvgoObject;
+};
+
+export const generate_random_id = string_length => {
```

Comment:
where is this function being used?

Reply:
I removed export since its only used inside this file